### PR TITLE
script offices: update of november arrivals and departures

### PR DESCRIPTION
**Pull request by TLasguignes**
Update the offices informations with latest arrivals and departures.
modification on the labels' creation to better manage the rooms with 5 persons.

### CODE REVIEW

File: scripts/offices.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 from wand.color import Color
 from wand.drawing import Drawing
 from wand.image import Image
+from wand.display import display
```
Comment (nim65s) [inline scripts/offices.py:16]: is this useful ?

File: scripts/offices.py [hunk 2]
```diff
@@ -109,7 +109,7 @@ def door_label(members, logo=True):
             with Image(filename=LOGO) as logo:
                 logo.transform(resize=f'{WIDTH}x{HEIGHT}')
                 draw.composite('over', 200, 0, logo.width, logo.height, logo)
-        draw.font_size = 80 if len(members) >= 4 else 90
+        draw.font_size = 70 if len(members) > 4 else (80 if len(members) == 4 else 90)
```
Comment (nim65s) [inline scripts/offices.py:112]: This start to be harder to read. I suggest the following:
```suggestion
        if len(members) > 4:
            draw.font_size = 70
        elif len(members) == 4:
            draw.font_size = 80
        else:
            draw.font_size = 90
```

File: scripts/offices.py (hunk 1 above)
Comment (TLasguignes) [reply scripts/offices.py:16]: This is no more useful. It has been forgotten while cleaning.

File: scripts/offices.py (hunk 2 above)
Comment (TLasguignes) [reply scripts/offices.py]: Ok.

**Comment by TLasguignes** [comment]
@nim65s I have applied your comments.

**Pull request merged**